.vscode/cspell.json

```diff
@@ -56,7 +56,6 @@
     "sdk/eventhub/azure-eventhub-checkpointstoretable/**",
     "sdk/eventgrid/azure-eventgrid/**",
     "sdk/eventhub/azure-eventhub-checkpointstoreblob-aio/**",
-    "sdk/eventhub/azure-eventhub-checkpointstoreblob/**",
```

Comment:
I would assume that both of these packages have similar spelling issues. If you're so inclined, would you be able to fix spelling in the aio package as well? 🙂 (If not, totally fine and we can fix this later!)

It also looks like you fixed spelling errors in our vendored storage folder (under `azure-eventhub-checkpointstoreblob/azure/eventhub/extensions/checkpointstoreblob/_vendor/storage/blob/`). We actually copied this directly from the [`azure-storage-blob`](https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/storage/azure-storage-blob) package at some point. The storage team might have fixed these typos in more recent versions of their package. However, if they didn't, would you be able to file another issue/PR for the storage-blob package instead? We can then pick up these changes from them next time we copy the vendored code over.

And here we can ignore spelling issues in the vendored code by replacing these lines with:
```suggestion
    "sdk/eventhub/azure-eventhub-checkpointstoreblob-aio/azure/eventhub/extensions/checkpointstoreblobaio/_vendor/**",
    "sdk/eventhub/azure-eventhub-checkpointstoreblob/azure/eventhub/extensions/checkpointstoreblob/_vendor/**",
```